environment.yml

```diff
@@ -10,27 +10,27 @@ channels:
 
 dependencies:
   - aiohttp
-  - cartopy <0.24  # https://github.com/ESMValGroup/ESMValTool/issues/3767
+  - cartopy
   - cdo >=2.3.0
   - cdsapi
   - cf-units
   - cfgrib
   - cftime
   - cmocean
-  - curl <8.10
+  - curl <8.10  # https://github.com/ESMValGroup/ESMValTool/issues/3758
```

Comment:
Why don't we have curl in the OSX env and setup.py file?

Reply:
because we don't do Julia stuff for OSX, Manu